packages/web-frontend/src/containers/MapOverlayBar/Control.js

```diff
@@ -71,12 +93,37 @@ const StyledOptionComponent = styled(StyledPrimaryComponent)`
   margin: -0.1rem 0;
 `;
 
+const LayersIcon = styled(MuiLayersIcon)`
+  font-size: 20px;
+  margin-left: 11px;
+  margin-right: 6px;
+  color: ${({ $expanded }) => ($expanded ? TUPAIA_ORANGE : 'default')};
+`;
+
+const DownArrowIconWrapper = styled.div`
+  display: flex;
+  padding: 8px 14px 0 5px;
+  .MuiSvgIcon-root {
+    transition: transform 0.3s ease;
+    transform: rotate(${({ expanded }) => (expanded ? '180deg' : '0deg')});
+  }
+
+  &:hover: {
```

Comment:
```suggestion
  &:hover {
```
Can you test this. I think the extra colon is a typo